### PR TITLE
Added events component

### PR DESCRIPTION
**Pull request by SingularReza**
Completed the event component with placeholder values

### CODE REVIEW

File: config/index.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ module.exports = {
 
     // Paths
     assetsSubDirectory: 'static',
-    assetsPublicPath: '/',
+    assetsPublicPath: 'lugnitdgp-website/',
```
Comment (ankan17) [inline config/index.js]: Remove this change

File: index.html [hunk 2]
```diff
@@ -3,10 +3,15 @@
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.2/css/bootstrap.min.css" integrity="sha384-Smlep5jCw/wG7hdkwQ/Z5nLIefveQRIY9nfy6xoR1uRYBtpZgI6339F5dgvm/e9B" crossorigin="anonymous">
```
Comment (ankan17) [inline index.html]: Do a rebase with the current master branch i.e., `git pull upstream master --rebase`. There will be some conflicts. Resolve those changes. Prefer the changes made on the master branch over your changes.

File: config/index.js (hunk 1 above)
Comment (SingularReza) [reply config/index.js]: removed it

**Comment by Compro-Prasad** [comment]
What is blocking this PR from being merged?

**Comment by ankan17** [comment]
Closing due to lack of response

**Pull request closed without merging**